### PR TITLE
3061 extend support to 2013

### PR DESCRIPTION
**Pull request by david-hm-morgan**
Adapt to a list of supported namespaces and iterate through them.

### CODE REVIEW

File: lib/text/ttml_text_parser.js [hunk 1]
```diff
@@ -1243,12 +1250,14 @@ shaka.text.TtmlTextParser.styleNs_ = 'http://www.w3.org/ns/ttml#styling';
 shaka.text.TtmlTextParser.styleEbuttsNs_ = 'urn:ebu:tt:style';
 
 /**
- * The namespace URL for SMPTE fields.
- * @const {string}
+ * The supported namespace URLs for SMPTE fields.
+ * @const Array.<{string}>
```
Comment (michellezhuogg) [inline lib/text/ttml_text_parser.js:1254]: {!Array.\<string\>}

File: lib/text/ttml_text_parser.js [hunk 2]
```diff
@@ -1243,12 +1250,14 @@ shaka.text.TtmlTextParser.styleNs_ = 'http://www.w3.org/ns/ttml#styling';
...
  * @private
  */
-shaka.text.TtmlTextParser.smpteNs_ =
-    'http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt';
+shaka.text.TtmlTextParser.smpteNs_ = [
```
Comment (michellezhuogg) [inline lib/text/ttml_text_parser.js:1257]: Would it be better to name it smpteNsList_ or something similar to indicate it's an array rather than a string?

**Review by michellezhuogg — COMMENTED** [review]
Thank you!

File: lib/text/ttml_text_parser.js (hunk 1 above)
Comment (david-hm-morgan) [reply lib/text/ttml_text_parser.js]: thanks, resolved in a fresh commit

File: lib/text/ttml_text_parser.js (hunk 2 above)
Comment (david-hm-morgan) [reply lib/text/ttml_text_parser.js]: Done

File: lib/text/ttml_text_parser.js [hunk 3]
```diff
@@ -1243,12 +1250,14 @@ shaka.text.TtmlTextParser.styleNs_ = 'http://www.w3.org/ns/ttml#styling';
 shaka.text.TtmlTextParser.styleEbuttsNs_ = 'urn:ebu:tt:style';
 
 /**
- * The namespace URL for SMPTE fields.
- * @const {string}
+ * The supported namespace URLs for SMPTE fields.
+ * @const !{Array.<{string}>}
```
Comment (michellezhuogg) [inline lib/text/ttml_text_parser.js:1254]: Style nit: {!Array.<string>}

Looks good otherwise. Thank you!

Comment (david-hm-morgan) [reply lib/text/ttml_text_parser.js]: doh!

**Review by michellezhuogg — APPROVED** [review]

**Comment by marvol120-dot-com** [comment]
> Adapt to a list of supported namespaces and iterate through them.

**Pull request merged**